packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/getting-started.doc.ts

```diff
@@ -0,0 +1,140 @@
+import type {LandingTemplateSchema} from '@shopify/generate-docs';
+
+const examplePath = './examples/getting-started';
+
+const data: LandingTemplateSchema = {
+  title: 'Getting started with POS UI extensions',
+  description:
+    'Learn how to prepare your development environment to start building POS UI extensions.',
+  id: 'getting-started',
```

Comment:
I think we can use this `id` to specify the order of the side bar. Let me look around too see how we can do this

Reply:
Gonna merge this, and then we can mess with the side bar later! mostly focusing on getting things moved to the new format for now. lmk, i can always change this for my next PR.